File: Prak504/PRAK504-2210817210021-TRISNACAHYAPERMADI.py
def reversed (reverse):
    n = 0
    while (reverse) :
        n = n*10 
        n = n + reverse%10
        reverse //=10
    return n
A, B = input().split()
A = reversed (int (A))
B = reversed (int (B))
C = A + B
print (reversed(C))

File: Prak504/PRAK504-2210817210021-TRISNACAHYAPERMADI.c
#include <stdio.h>
int reverse (int A){
    int reverse = 0;
    while (A!=0){
        reverse = reverse * 10;
        reverse = reverse + A%10;
        A = A/10;
    }
     return reverse; 
}
int main (){
    int A, B;
    scanf ("%i %i", &A, &B);
    A = reverse (A);
    B = reverse (B);
    int C = A + B;
    printf ("%i", reverse(C));
}
